descartes_light/core/include/descartes_light/solvers/ladder_graph/ladder_graph.h

```diff
@@ -26,6 +26,7 @@ DESCARTES_IGNORE_WARNINGS_PUSH
 #include <vector>
 #include <Eigen/Geometry>
 #include <iomanip>
+#include <fstream>
```

Comment:
This can be moved to the `ladder_graph.hpp` and replace here with the the `#include <iosfwd>`